web/themes/custom/cloudy/src/components/badge/badge.scss

```diff
@@ -30,53 +30,42 @@
   }
 
   &--info {
-    color: $cloudy-color-blue-800;
-    background-color: $cloudy-color-blue-100;
+    color: $cloudy-color-blue-700;
+    background-color: $cloudy-color-blue-200;
 
     &.cloudy-badge--bold {
       color: $cloudy-color-neutral-0;
-      background-color: $cloudy-color-blue-800;
+      background-color: $cloudy-color-blue-600;
     }
   }
 
   &--success {
-    color: $cloudy-color-green-500;
-    background-color: $cloudy-color-green-100;
+    color: $cloudy-color-green-800;
+    background-color: $cloudy-color-green-200;
 
     &.cloudy-badge--bold {
       color: $cloudy-color-neutral-0;
-      background-color: $cloudy-color-green-500;
+      background-color: $cloudy-color-green-700;
     }
   }
 
   &--warning {
     color: $cloudy-color-neutral-900;
-    background-color: $cloudy-color-yellow-100;
+    background-color: $cloudy-color-yellow-200;
 
     &.cloudy-badge--bold {
       color: $cloudy-color-neutral-900;
-      background-color: $cloudy-color-yellow-800;
+      background-color: $cloudy-color-yellow-500;
     }
   }
 
   &--danger {
-    color: $cloudy-color-red-500;
-    background-color: $cloudy-color-red-100;
+    color: $cloudy-color-red-700;
+    background-color: $cloudy-color-red-200;
 
     &.cloudy-badge--bold {
       color: $cloudy-color-neutral-0;
-      background-color: $cloudy-color-red-500;
+      background-color: $cloudy-color-red-600;
     }
   }
 }
-
-/* admin content/media view tables */
-.view-admin-group-content .status-badge {
-  margin-right: 5px;
-}
-.status-unpublished {
-  color: $cloudy-color-neutral-100;
-}
-.status-published {
```

Comment:
I moved these classes over to cloudy/src/components/_badge.scss since they aren't part of the badge component. The classes are being applied exactly as before; however, I did notice that the color updates lighten the previously set color variables. I will update and add to the test plan.